frontend/src/components/ExperimentCollection/ExperimentCollection.jsx

```diff
@@ -1,39 +1,39 @@
-import { Link, Redirect } from "react-router-dom";
-
+import {
+    BrowserRouter as Router,
+    Route,
+    Redirect
+} from "react-router-dom";
 import { useExperimentCollection } from "../../API";
 import Loading from "../Loading/Loading";
-import { API_ROOT } from "../../config";
+import ExperimentCollectionAbout from "./ExperimentCollectionAbout/ExperimentCollectionAbout";
+import ExperimentCollectionDashboard from "./ExperimentCollectionDashboard/ExperimentCollectionDashboard";
+import { URLS } from "../../config";
 
 const ExperimentCollection = ({ match }) => {
-        /** ExperimentCollection is a Component which can show a dashboard of multiple experiments,
-     * or redirect to the next experiment in the collection,
-     * depending on the response from the backend, which has the following structure: 
-     * {
-     *   slug: string,
-     *   name: string,
-     *   description: string,
-     *   dashboard: [
-     *     {
-     *       slug: string,
-     *       name: string,
-     *       finished_session_count: number
-     *     }
-     *   ],
-     *   redirect_to: {
-     *     slug: string,
-     *     name: string,
-     *     finished_session_count: number
-     *   }
-     * }
-     * */ 
+    /** ExperimentCollection is a Component which can show a dashboard of multiple experiments,
+ * or redirect to the next experiment in the collection,
+ * depending on the response from the backend, which has the following structure: 
+ * {
+ *   slug: string,
+ *   name: string,
+ *   description: string,
+ *   dashboard: [
+ *     {
+ *       slug: string,
+ *       name: string,
+ *       finished_session_count: number
+ *     }
+ *   ],
+ *   redirect_to: {
+ *     slug: string,
+ *     name: string,
+ *     finished_session_count: number
+ *   }
+ * }
+ * */
```

Comment:
perhaps this docstring can be removed, as the interface documents this now.

Reply:
I could do this and maybe straight away convert the whole file to `.tsx`? Then I can type the props as well.